modules/nw-configure-sysctl-interface-sriov-network.adoc

```diff
@@ -0,0 +1,171 @@
+spec:
+  resourceName: policy-example-oneflag <3>
+  networkNamespace: sysctl-tuning-test <4>
+  ipam: '{ "type": "static" }' <5>
+  capabilities: '{ "mac": true, "ips": true }' <6>
+  metaPlugins : | <7>
+    {
+      "type": "tuning",
+      "capabilities":{
+        "mac":true
+      },
+      "sysctl":{ 
+         "net.ipv4.conf.IFNAME.accept_redirects": "1"
+      }
+    }
+----
+<1> A name for the object. The SR-IOV Network Operator creates a NetworkAttachmentDefinition object with same name.
+<2> The namespace where the SR-IOV Network Operator is installed.
+<3> The value for the spec.resourceName parameter from the SriovNetworkNodePolicy object that defines the SR-IOV hardware for this additional network.
+<4> The target namespace for the SriovNetwork object. Only pods in the target namespace can attach to the additional network.
+<5> A configuration object for the IPAM CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
+<6> Optional: The capabilities to configure for this additional network. You can specify "{ "ips": true }" to enable IP address support or "{ "mac": true }" to enable MAC address support.
+<7> Optional: The metaPlugins parameter is used to add additional capabilities to the device. In this use case set the `type` field to `tuning`. Specify the interface-level network `sysctl` you want to set in the `sysctl` field. 
+
+. Create the `SriovNetwork` resource:
++
+[source,terminal]
+----
+$ oc create -f sriov-network-interface-sysctl.yaml
+----
+
+.Verifying that the `NetworkAttachmentDefinition` CR is successfully created
+
+* Confirm that the SR-IOV Network Operator created the `NetworkAttachmentDefinition` CR by running the following command.
++
+[source,terminal]
+----
+$ oc get network-attachment-definitions -n <namespace> <1>
+----
+<1> Replace `<namespace>` with the namespace that you specified when configuring the network attachment, for example, `openshift-sriov-network-operator`.
++
+.Example output
+[source,terminal]
+----
+NAME                                  AGE
+openshift-sriov-network-operator      14m
+----
++
+[NOTE]
+====
+There might be a delay before the SR-IOV Network Operator creates the CR.
+====
+
+.Verifying that the additional SR-IOV network attachment is successful
+
+To verify that the tuning CNI is correctly configured and the additional SR-IOV network attachment is attached, do the following:
+
+. Create a pod such as `examplepod.yaml` similar to the following:.
++
+[source,terminal]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: tunepod
+  namespace: default
+  annotations:
+    k8s.v1.cni.cncf.io/networks: |-
+      [
+        { 
+          "name": "onevalidflag",  <1>
+          "mac": ["0a:56:0a:83:04:0c"], <2>
+          "ips": ["10.100.100.200/24"]'<3>
+       } 
+      ]
+spec:
+  containers:
+  - name: podexample
+    image: centos
+    command: ["/bin/bash", "-c", "sleep INF"]
+    securityContext:
```

Comment:
Having the securityContext might hint that some additional privileges are needed.
Do we need this?

Reply:
Hi @mmirecki I used file that @kononovn referred to. I think might be needed to deal with security updates in 4.11.

Reply:
If I run without securityContext I get the error 
```
oc apply -f example3.yaml
Warning: would violate PodSecurity "restricted:v1.24": allowPrivilegeEscalation != false (container "podexample" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "podexample" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "podexample" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "podexample" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
pod/tunepod created
```

Reply:
When I use example pod supplied by @kononovn it does not dump out that error